geofeat/features.py

```diff
@@ -37,7 +37,8 @@ def __init__(self, db, df, long_column, lat_column, id_column='sys_ind'):
         self.cache = (0, '')
 
     def increment(self, df):
-        return df.reset_index().rename(columns={'index': self.id_column})
+        index = 'index' if df.index.name is None else df.index.name
```

Comment:
Why not just leaving index = df.index.name? Isn't it returns 'index' string in case when index name not specified?